service-base/src/main/java/org/eclipse/hono/service/http/HttpServiceConfigProperties.java

```diff
@@ -26,10 +26,14 @@ public class HttpServiceConfigProperties extends ServiceConfigProperties {
      * The default name of the realm that clients need to authenticate to.
      */
     public static final String DEFAULT_REALM = "Hono";
+    /**
+     * The default idle timeout.
+     */
+    public static final int DEFAULT_IDLE_TIMEOUT_SECONDS = 65;
```

Comment:
FMPOV there is no need to change this value because the registry does not need to consider a TTD parameter, does it?

Reply:
Right, there is no such need. I only thought to use the same value throughout Hono. But I'm also fine with using the higher value only with the HTTP adapter, emphasizing that there are special considerations here.